.github/workflows/build.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           kubectl config use-context ci
 
       - run: tilt ci -- --ci --namespace=$DEPLOY_NS --num=2
-        timeout-minutes: 20
+        timeout-minutes: 40
```

Comment:
don't love this, but as long as we can bring this back down with optimizations we can live with it being higher temporarily (I hope)

Reply:
CI will definitely run pretty quick if the ethereum folder / relayer engine code aren't touched (because the docker images are cached)

If the ethereum folder is touched, now (after all the generic relayer changes) CI will be 6 minutes slower than it would have been before, since we compile contracts in CI using via-ir. 

If the relayer engine is touched, in the current moment there is a significant CI slowdown (at least 6 if not more minutes) because the docker image has to be re-built meaning the ethereum folder (as a dependency) will have to be rebuilt. However, once the change of importing the SDK is made, this slowdown will be much less significant because the relayer engine will no longer have /ethereum as a dependency.